src/inc/msquic.h

```diff
@@ -110,6 +110,7 @@ typedef enum QUIC_CREDENTIAL_FLAGS {
     QUIC_CREDENTIAL_FLAG_SET_ALLOWED_CIPHER_SUITES              = 0x00002000,
     QUIC_CREDENTIAL_FLAG_USE_PORTABLE_CERTIFICATES              = 0x00004000,
     QUIC_CREDENTIAL_FLAG_USE_SUPPLIED_CREDENTIALS               = 0x00008000, // Schannel only currently
+    QUIC_CREDENTIAL_FLAG_INDICATE_NULL_CLIENT_CERTIFICATE       = 0x00010000,
```

Comment:
```suggestion
    QUIC_CREDENTIAL_FLAG_INDICATE_NULL_CERTIFICATE                    = 0x00010000,
```
I think we don't need to include CLIENT in the name. BTW, we need to add validation logic to fail with invalid parameter if this is passed on client.

Reply:
After discussion, removing this flag and making the existing flags work.